prime-router/src/main/kotlin/credentials/CredentialManagement.kt

```diff
@@ -6,6 +6,13 @@ class CredentialHelper() {
         fun getCredentialService(): CredentialService {
             return credentialServiceForStorageMethod()
         }
+
+        fun formCredentialLabel(fromReceiverName: String): String {
```

Comment:
As a little bit of clean-up, I'm putting this naming recipe behind a function and removing the recipe that is scattered in our code.